guides/doc-Upgrading_and_Updating/topics/proc_synchronizing-smartproxy-servers-through-project-web-ui.adoc

```diff
@@ -0,0 +1,10 @@
+[id="proc_synchronizing-{smart-proxy-context}-servers-through-{project-context}-web-ui_{context}"]
```

Comment:
Pls remove `proc_` from the ID and in the xref below. (Keep it in the file name.)

Reply:
I'd prefer this one to be applied.

Reply:
Applied.